packages/notification/test/animation.test.js

```diff
@@ -1,6 +1,5 @@
 import { expect } from '@esm-bundle/chai';
 import { aTimeout, fixtureSync, oneEvent } from '@vaadin/testing-helpers';
-import '@vaadin/polymer-legacy-adapter/template-renderer.js';
```

Comment:
Let's update `package.json` accordingly if the dev dependency is no longer used.

Reply:
There's still the `template.test.js`, which explicitly tests the `<template>` API...wasn't sure if we want to remove that yet. wdyt?

Reply:
Oh, right. Maybe we could move it to `integration` package, but let's consider doing that separately.